js/reference.js

```diff
@@ -166,6 +167,24 @@ var ERMrest = (function(module) {
         },
 
         /**
+         * The session object from the server
+         * @param {Object} session - the session object
+         */
+        set session(session) {
```

Comment:
It seems strange to add a useless getter (and then tell people not to use it) to avoid a lint warning. Would it be preferable to tell jshint to ignore this specific warning for this line instead? i.e. 

```
/* jshint -W078 */ //Tells jshint to not lint for warning #078 starting at this line
set session(session) {
    ...
},
/*jshint +W078 */ //Tells jshint to resume linting for warning #078 starting at this line
```

And then remove the getter.